src/components/UI/dropdown/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState, useRef } from 'react';
+import Link from 'next/link';
+import styles from './dropdown.module.scss';
+
+const Dropdown = ({
+  isLoggedIn,
+  USER_PROFILE_URL,
+  SIGN_OUT_URL,
```

Comment:
should be coming directly from the constants file, why are we passing it as a prop?

Reply:
Because I tried to make the component re-usable.